authorization.md

```diff
@@ -800,3 +729,45 @@ When attempting to determine if the authenticated user can update a given post,
 
         return redirect('/posts');
     }
+
+<a name="authorization-and-inertia"></a>
+## Authorization & Inertia
```

Comment:
@taylorotwell nice solution, thanks for taking my suggestions and building on it :+1: